tests/remove-data-test-id.spec.js

```diff
@@ -110,6 +110,14 @@ const runTests = (label, transform) => {
           const expected = transform(expectedCode);
           expect(uglify(actual)).to.equal(uglify(expected));
         });
+
+        it('removes options.attributes null', () => {
```

Comment:
👍

Reply:
I don' t think this solved the test bc I was able to add that to `master` and the test had passed without my changes. I need to look into this more rq before its merged.

Reply:
Ok, no worries - ping me when ready

Reply:
Should be working now. Updated original issue with the failing jsx.

Reply:
Sorry just got back online, seems good - going to go through publish now